ontopy/utils.py

```diff
@@ -94,12 +97,45 @@ def isinteractive():
     )
 
 
+def get_preferred_language(langstrings: list) -> str:
+    """Given a list of localised strings, return the one for the
+    preferred language.  If no one match the preferred language,
+    return the one with no language tag or fallback to the first
+    string.
+
+    The preferred language is stored as a module variable. You can
+    change it with:
+
+    >>> import ontopy.utils
+    >>> ontopy.utils.PREFERRED_LANGUAGE = "en"
+
+    """
+    for langstr in langstrings:
+        if hasattr(langstr, "lang") and langstr.lang == PREFERRED_LANGUAGE:
+            return langstr
+    for langstr in langstrings:
+        if not hasattr(langstr, "lang"):
+            return langstr
+    return langstr[0]
```

Comment:
This is tested very indirectly. Isnæt it better to test this in a unit test?

Reply:
Added explicit test